src/main/java/service/fileService/DataVerticalWriterImpl.java

```diff
@@ -0,0 +1,35 @@
+package service.fileService;
+
+import com.opencsv.bean.HeaderColumnNameMappingStrategy;
+import com.opencsv.bean.HeaderColumnNameMappingStrategyBuilder;
+import com.opencsv.bean.StatefulBeanToCsv;
+import com.opencsv.bean.StatefulBeanToCsvBuilder;
+import com.opencsv.exceptions.CsvDataTypeMismatchException;
+import com.opencsv.exceptions.CsvRequiredFieldEmptyException;
+import model.Blind;
+import model.BlindVertical;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+public class DataVerticalWriterImpl implements DataWriter {
+    @Override
+    public void writeDataToFile(List<? extends Blind> blindVerticalList) {
```

Comment:
Let's move the same logic for this method and method from another writeDataToFile() to the parent class